staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/conversion.go

```diff
@@ -65,7 +65,7 @@ func Convert_v1_JSON_To_apiextensions_JSON(in *JSON, out *apiextensions.JSON, s
 		}
 		*out = i
 	} else {
-		out = nil
+		*out = nil
```

Comment:
this is a really subtle one, we'll need to tag in API reviewers

Reply:
Without this:

```
ERROR: staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/conversion.go:68:3: ineffectual assignment to out (ineffassign)
ERROR: 		out = nil
ERROR: 		^
```